ca_description/urdf/create_apriltag.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+<xacro:macro name="create_apriltag">
+ <joint name="apriltag_joint" type="fixed">
+    <origin xyz="0 0 0.25" rpy="0 0 0" />
+    <parent link="base_link" />
+    <child link="apriltag_link" />
+  </joint>
+
+  <link name="apriltag_link">
+    <xacro:dummy_inertia/>
```

Comment:
Is the inertia necessary? Given that you're attached by a fixed link, I don't think so. Please, check.